### DISCUSSION TITLE
Where is 'Discussion answered' highlight?

### DISCUSSION THREAD

**Discussion by rodukov** [community/community#31629] [Other Feature Feedback, Questions, & Ideas] [answered]
**https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile**
I noticed that the number of replies in my profile was gone. **However, there is still a link to this highlight in the documentation.**
I'm wondering if it's possible to bring it back.
![img](https://user-images.githubusercontent.com/101352977/187961868-4e71ad28-dce0-4bc7-9686-fd662a861711.png)

**Comment by radar-blip** [comment]
Github disabled that for some reason, not sure why. 
Maybe because its also visible in your contribution activity.
Time will tell.

**Comment by jfullstackdev** [comment]
I think the same thing: because the badge is being abused

**Comment by melvincwng** [comment, marked as answer]
Hi @rodukov 👋, the 'Discussions answered' badge/highlight has been replaced with the Galaxy Brain achievement.

I previously contacted GitHub Support, and one of their support engineers mentioned to me that this was an intended change (remove 'Discussions answered' badge & replace it with 'Galaxy Brain' achievement).

I understand that there was a period of time where both the Discussions answered badge and the Galaxy Brain achievement could be found in a user's profile. But unfortunately that is no longer the case.

So unfortunately we won't be getting back the Discussions Answered highlight/badge.

Hope this clarifies and answers your question 😊
